myapp/form.py

```diff
@@ -0,0 +1,20 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+# from django.views.decorators.csrf import csrf_protect
+
+# Create your forms here.
+# @csrf_protect
+class NewUserForm(UserCreationForm):
+	email = forms.EmailField(required=True)
+
+	class Meta:
+		model = User
+		fields = ("username", "email", "password1", "password2")
+
+	def save(self,commit=True):
+		user = super(NewUserForm, self).save(commit=False)
+		user.email = self.cleaned_data['email']
+		if commit:
+			user.save()
```

Comment:
Please maintains the identation everywhere. Please be careful when you do the code. This is the repeated comment.